### ISSUE TITLE
More basic Image processing tutorials

### ISSUE THREAD

**Issue by take2rohit** [Tessellate-Imaging/monk_v1#111]
Your Image processing basics are good. But do you think that we should add some more even basic tutorials? Like adding of two image, subtracting of 2 image, multiplication of two image, etc?  I have a notebook for this i can modify it and incase u want i can create a PR also,

[Link](https://github.com/Tessellate-Imaging/monk_v1/tree/master/study_roadmaps/3_image_processing_deep_learning_roadmap/1_image_processing_basics])

**Comment by abhi-kumar** [comment]
Yes, we are planning to add more tutorial to it to make it more complete. Do add your contribution to monk, we will reformat it accordingly.

**Comment by take2rohit** [comment]
Ok. I will make a PR by tomorrow.

Also one suggestion could be can u please add labels in issues as a part of hacktober fest? This will attract more devs. You may also need to add hacktober fest as title of the repo (for a month maybe). Thanks

Info: [LINK](https://hacktoberfest.digitalocean.com/hacktoberfest-update)

**Comment by abhi-kumar** [comment]
I have added hacktoberfest label for PRs/Issues as well as added it to the title-tags

**Comment by take2rohit** [comment]
I have gone through the notebooks [here](https://github.com/Tessellate-Imaging/monk_v1/tree/master/study_roadmaps/3_image_processing_deep_learning_roadmap/1_image_processing_basics). It seems that basic manipulations are already covered. Incase you have some other ideas to work on, let me know. I can work on that maybe.

**Comment by abhi-kumar** [comment]
Sure, will let you know as soon as we have a plan for more tutorials. Do share if you too have something in mind regarding the same.
